app/src/lib/editors/win32.ts

```diff
@@ -10,458 +10,279 @@ import {
+    executableShimPath: ['subl.exe'],
+    expectedInstallationChecker: (displayName, publisher) =>
+      displayName.startsWith('Sublime Text') &&
+      publisher === 'Sublime HQ Pty Ltd',
+  },
+  {
+    name: 'ColdFusion Builder',
+    registryKeys: [
+      // 64-bit version of ColdFusionBuilder3
+      LocalMachineUninstallKey('Adobe ColdFusion Builder 3_is1'),
+      // 64-bit version of ColdFusionBuilder2016
+      LocalMachineUninstallKey('Adobe ColdFusion Builder 2016'),
+    ],
+    executableShimPath: ['CFBuilder.exe'],
+    expectedInstallationChecker: (displayName, publisher) =>
+      displayName.startsWith('Adobe ColdFusion Builder') &&
+      publisher === 'Adobe Systems Incorporated',
+  },
+  {
+    name: 'Typora',
+    registryKeys: [
+      // 64-bit version of Typora
+      LocalMachineUninstallKey('{37771A20-7167-44C0-B322-FD3E54C56156}_is1'),
+      // 32-bit version of Typora
+      Wow64LocalMachineUninstallKey(
+        '{37771A20-7167-44C0-B322-FD3E54C56156}_is1'
+      ),
+    ],
+    executableShimPath: ['typora.exe'],
+    expectedInstallationChecker: (displayName, publisher) =>
+      displayName.startsWith('Typora') && publisher === 'typora.io',
+  },
+  {
+    name: 'SlickEdit',
+    registryKeys: [
+      // 64-bit version of SlickEdit Pro 2018
+      LocalMachineUninstallKey('{18406187-F49E-4822-CAF2-1D25C0C83BA2}'),
+      // 32-bit version of SlickEdit Pro 2018
+      Wow64LocalMachineUninstallKey('{18006187-F49E-4822-CAF2-1D25C0C83BA2}'),
+      // 64-bit version of SlickEdit Standard 2018
+      LocalMachineUninstallKey('{18606187-F49E-4822-CAF2-1D25C0C83BA2}'),
+      // 32-bit version of SlickEdit Standard 2018
+      Wow64LocalMachineUninstallKey('{18206187-F49E-4822-CAF2-1D25C0C83BA2}'),
+      // 64-bit version of SlickEdit Pro 2017
+      LocalMachineUninstallKey('{15406187-F49E-4822-CAF2-1D25C0C83BA2}'),
+      // 32-bit version of SlickEdit Pro 2017
+      Wow64LocalMachineUninstallKey('{15006187-F49E-4822-CAF2-1D25C0C83BA2}'),
+      // 64-bit version of SlickEdit Pro 2016 (21.0.1)
+      LocalMachineUninstallKey('{10C06187-F49E-4822-CAF2-1D25C0C83BA2}'),
+      // 64-bit version of SlickEdit Pro 2016 (21.0.0)
+      LocalMachineUninstallKey('{10406187-F49E-4822-CAF2-1D25C0C83BA2}'),
+      // 64-bit version of SlickEdit Pro 2015 (20.0.3)
+      LocalMachineUninstallKey('{0DC06187-F49E-4822-CAF2-1D25C0C83BA2}'),
+      // 64-bit version of SlickEdit Pro 2015 (20.0.2)
+      LocalMachineUninstallKey('{0D406187-F49E-4822-CAF2-1D25C0C83BA2}'),
+      // 64-bit version of SlickEdit Pro 2014 (19.0.2)
+      LocalMachineUninstallKey('{7CC0E567-ACD6-41E8-95DA-154CEEDB0A18}'),
+    ],
+    executableShimPath: ['win', 'vs.exe'],
+    expectedInstallationChecker: (displayName, publisher) =>
+      displayName.startsWith('SlickEdit') && publisher === 'SlickEdit Inc.',
+  },
+  {
+    name: 'JetBrains Webstorm',
+    registryKeys: [
+      // Webstorm 2018.3
```

Comment:
Given that the uninstall key match the comment exactly I don't feel like these comments add much value

```suggestion
```